src/Components/Navbar/Navbar.css

```diff
@@ -0,0 +1,79 @@
+.top-bar
+{
+    z-index: 10;
+    border: none;
+    background-color: white;
+    padding: 2rem 4rem;
+    color: var(--background-color);
+    border-bottom: 1px solid var(--background-color);
+}
+
+@media screen and (max-width:1305px){
+    .top-bar
```

Comment:
Suggestion: Use @media only screen

Reply:
Nice point, changed, thanks for point out.